packages/manager/apps/container/src/container/legacy/server-sidebar/order/shop-config/order.constants.ts

```diff
@@ -409,6 +423,15 @@ export const ORDER_URLS: Record<
       WE: 'https://www.ovhcloud.com/en/managed-bare-metal/',
       WS: 'https://www.ovhcloud.com/es/managed-bare-metal/',
     },
+    sap_hana: {
+      QC: 'https:/www.ovhcloud.com/fr-ca/hosted-private-cloud/sap-hana/',
+      WS: 'https:/www.ovhcloud.com/es/hosted-private-cloud/sap-hana/',
+      WE: 'https:/www.ovhcloud.com/en/hosted-private-cloud/sap-hana/',
+      SG: 'https:/www.ovhcloud.com/en-sg/hosted-private-cloud/sap-hana/',
+      CA: 'https:/www.ovhcloud.com/en-ca/hosted-private-cloud/sap-hana/',
+      AU: 'https:/www.ovhcloud.com/en-au/hosted-private-cloud/sap-hana/',
+      ASIA: 'https:/www.ovhcloud.com/asia/hosted-private-cloud/sap-hana/',
```

Comment:
You can add IN: 'https:/www.ovhcloud.com/asia/hosted-private-cloud/sap-hana/' as well. It avoids re work.

Reply:
It's applied.